scripts/gen-service-kustomize.sh

```diff
@@ -64,6 +65,12 @@ patches:
     - op: replace
       path: /spec/storageClass
       value: ${STORAGE_CLASS}
+    - op: replace
+      path: /spec/galera/templates/openstack/storageRequest
```

Comment:
This is not good in case we deploy galera w/ `make mariadb_deploy`

Reply:
I think I solved this particular problem: the default `mariadb` samples are problematic with the `500M` pvc size, so I made it configurable.